node/out_response.js

```diff
@@ -193,6 +226,20 @@ TChannelOutResponse.prototype.sendError = function sendError(codeString, message
 
         self.codeString = codeString;
         self.message = message;
+        self.channel.inboundCallsSystemErrorsStat.increment(1, {
+            'calling-service': self.callingService,
+            'service': self.serviceName,
+            'endpoint': self.endpoint,
+            'type': self.codeString
+        });
+
+        if (self.codeString === 'ProtocolError') {
+            self.channel.inboundProtocolErrorsStat.increment(1, {
```

Comment:
This is not how the protocol error stat works; its for protocol errors like "init request failed".